### PR TITLE
docs(readme): add information on debugging releases and writing good commit messages

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Ticket Number

N/A

## Description of change

Update docs to help finding issues with release management

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [x] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,59 @@ variables which we want to apply (in this instance, just the URL).
...
+and one `feat` will produce a new release bumping the minor semantic version number.
+Without the `feat`, it would create no new release.
+
+## Checking The Correct Release Has Been Deployed
+
+1. Check your PR has passed. If there are any failures, check these to see if the
+reasons for failure give a clue as to what went wrong (and then fix). There is a job
+called `Next version` which will tell you the version number that this should create
+if successful.
+2. Check a new release was made. Dependent upon whether it was made from  the `develop`
```
Comment (a6software) [inline README.md:145]: where would we look? - assume here - https://github.com/foundry4/appeal-planning-decision/releases

File: README.md [hunk 2]
```diff
@@ -116,7 +116,59 @@ variables which we want to apply (in this instance, just the URL).
...
+
+1. Check your PR has passed. If there are any failures, check these to see if the
+reasons for failure give a clue as to what went wrong (and then fix). There is a job
+called `Next version` which will tell you the version number that this should create
+if successful.
+2. Check a new release was made. Dependent upon whether it was made from  the `develop` 
+or `master` branch, you will be looking for either a pre-release version  or a release. 
+If no release has been made, ensure that your commit message was formatted  correctly 
+and begins with `feat` or `fix`.
+3. Check the `release` branch against the cluster you are expecting to see it deployed on.
```
Comment (a6software) [inline README.md:149]: not sure how to do this step

File: README.md (hunk 1 above)
Comment (MrSimonEmms) [reply README.md]: Correct. I've put a link in

**Review by danWatkinson — APPROVED** [review]
approving as 'acknowledgement of having read the changes'

File: README.md (hunk 2 above)
Comment (MrSimonEmms) [reply README.md]: I've put a link into the GitHub page. It's then go to the cluster and the `app.yml` file as (hopefully) described

**Comment by MrSimonEmms** [comment]
> approving as 'acknowledgement of having read the changes'

Six housepoints to @danWatkinson. Happy Christmas 🎅🏻

**Review by a6software — APPROVED** [review]

**Review by Fenisay — APPROVED** [review]

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by frank-dodion — APPROVED** [review]

**Review by karunak21 — APPROVED** [review]

**Review by GoEagles999 — APPROVED** [review]

**Review by imansell — APPROVED** [review]

**Pull request merged**